test/modules/geo-layers/tile-layer/tile-2d-header.spec.js

```diff
@@ -5,7 +5,7 @@ import {RequestScheduler} from '@loaders.gl/loader-utils';
 test('Tile2DHeader', async t => {
   let onTileErrorCalled = false;
 
-  const requestScheduler = new RequestScheduler({throttleRequests: false});
+  const requestScheduler = new RequestScheduler({maxRequests: -1});
```

Comment:
Oh here I'm instantiating the RequestScheduler directly, so I still need `throttleRequests: false`